pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -3293,6 +3298,57 @@ var _ = Context("Scheduling", func() {
 			// no nodes should be created as the storage class is using an unsupported provisioner
 			Expect(nodeList.Items).To(HaveLen(0))
 		})
+		It("should not launch nodes for pod with unbound volume for voluemBindingMode immediate", func() {
```

Comment:
Is it also possible to assert the metrics are what we expect?

Reply:
Good catch, done.